test/Dialect/FIRRTL/parse-name-preservation.fir

```diff
@@ -0,0 +1,22 @@
+; RUN: firtool --parse-only %s | FileCheck %s
```

Comment:
We don't have a way to plumb options through `circt-translate` right now.  Are there any name preservation unit tests somewhere I can add this to that I missed?  The existing tests I saw test the final output verilog.

Reply:
No, there are no tests of this at just the `circt-tranlsate` level for this reason. 😅